app/controllers/resources_controller.rb

```diff
@@ -36,6 +36,18 @@ def push_to_onesky
 
   private
 
+  def cached_index_json
+    cache_key = Resource.index_cache_key(all_resources, include: params[:include])
```

Comment:
needs to include the new sparse fieldset params

Reply:
you also should probably cache on any specified filters (assuming we are actually applying filters)

Reply:
anyway to serialize all of `params` into a string to avoid needing to account for every individual param in the cache key?